### ISSUE TITLE
the implementation of batch normalization in your model

### ISSUE THREAD

**Issue by azshue** [richzhang/colorization#62]
Hi,

I have a little question about the BatchNorm layer in you model. 

In ./colorization/models/colorization_train_val_v2.prototxt, I noticed that you are using `BatchNorm` layers without `Scale` layers following behind, so does this mean that you are not using the original definition of BatchNorm that has additional scaling factors and biases?

If my understanding is correct, have you tried BN layers with scaling factors and biases? I am wondering to what degree will this difference affects the training results.
